src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -0,0 +1,66 @@
+package juja.sqlcmd;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class DatabaseManagerTest {
+    private DatabaseManager databaseManager;
+    private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+    private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+    private PrintStream originalOut;
+    private PrintStream originalErr;
+
+    @Before
+    public void setUpStreamsAndDbManager() {
+        originalOut = System.out;
+        originalErr = System.err;
+        System.setOut(new PrintStream(outContent));
+        System.setErr(new PrintStream(errContent));
+        databaseManager = new DatabaseManager();
+    }
+
+    @After
+    public void cleanUpStreams() {
```

Comment:
You don't need this method after deleting unnesessary lines from setUpStreamsAndDbManager()